### PR TITLE
Frontend/reactions db interaction

### PR DESCRIPTION
**Pull request by Tenyama**
## Reaction now adds and deletes in the db 
image

### CODE REVIEW

File: client/src/types/post.ts [hunk 1]
```diff
@@ -29,17 +29,17 @@ export interface User {
   profileImage?: string;
 }
 
-export interface Reaction {
+export type Reaction = {
   author: User;
-  type: ReactionTypes;
-}
+  type: string;
```
Comment (Hankaji) [inline client/src/types/post.ts]: Why?

**Review by Hankaji — APPROVED** [review]
I hate you

File: client/src/types/post.ts (hunk 1 above)
Comment (Tenyama) [reply client/src/types/post.ts:34]: Backend: ReactionTypes: Like Love Angry Haha
Frontend: LIKE LOVE ANGRY HAHA
=> Must use toUppercase() which only works on String

Comment (Tenyama) [reply client/src/types/post.ts]: Mb bbg from my old code

**Pull request merged**